testsuite/libra-fuzzer/src/lib.rs

```diff
@@ -13,7 +13,13 @@ pub mod fuzz_targets;
 /// Implementation for a particular target of a fuzz operation.
 pub trait FuzzTargetImpl: Sync + Send + fmt::Debug {
     /// The name of the fuzz target.
-    fn name(&self) -> &'static str;
+    /// By default, we use the struct name, however, implementations may prefer to override this.
+    fn name(&self) -> &'static str {
+        std::any::type_name::<Self>()
+            .rsplit("::")
+            .next()
```

Comment:
don't you want last() ?

Reply:
rsplit does a reverse split 😃 https://doc.rust-lang.org/std/primitive.str.html#method.rsplit